package.json

```diff
@@ -91,7 +91,7 @@
         "webpack-dev-server": "^4.10.1"
     },
     "dependencies": {
-        "@aics/simularium-viewer": "^3.5.1",
```

Comment:
is this change "sneaking in" here?

Reply:
oops yes, bumping now

Reply:
what I meant is, changing the actual viewer version seems out of scope of this PR

Reply:
Hmm, not sure I'm understanding. We want this feature branch to reflect the same viewer version as `main`, yes? So I ran `npm run update-viewer` and committed after reading your comment. Sorry to make you spell this out.

Reply:
I think it's because this was branched from main a while ago. I would merge main here and this should stop showing in the diff.

Reply:
Since the base branch of this PR is main, I would not expect there to show a diff in the viewer version here.  I hope that merging main in to here might help resolve it.

Reply:
Merged main and now its showing 46 files changed, all sorts of stuff from the merge appearing in diff. I must be doing this wrong. 

Will try to revert and re-merge, but open to advice.